app/scraper.rb

```diff
@@ -39,7 +28,7 @@ def scrape_jobs
     html = self.class.get("#{BASE_URL}/")
 
     if html.body.nil? || html.body.empty?
```

Comment:
Use .to_s.empty?: This combines the nil check and the empty check into a single call. to_s converts nil to an empty string (""), and empty? checks whether the string is empty. This is a common Ruby idiom that simplifies checking for both nil and empty strings.

`if html.body.to_s.empty?`

Reply:
changed, also my bad, cuz i used the code httparty told me to use to check html body in his error message